lms/djangoapps/support/static/support/jsx/entitlements/components/Table/TableContainer.jsx

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+
+import EntitlementTable from './Table';
+
+const mapStateToProps = state => ({
+  entitlements: state.entitlements,
+});
+
+const TableContainer = connect(
+  mapStateToProps,
+  {},
```

Comment:
Does this work or does it need a function?

Reply:
it does work, in that it doesn't error out, I also don't need to pass it though I believe. connect can take many args and seems to sensibly default ([definition](https://github.com/reactjs/react-redux/blob/5d792a283554cff3d2f54fad1be1f79cbcab33fe/src/connect/connect.js#L47))

Reply:
I'll probably remove this for now.